src/pages/resultsView/plots/PlotsTab.tsx

```diff
@@ -1346,7 +1346,7 @@ export default class PlotsTab extends React.Component<IPlotsTabProps,{}> {
                             </Observer>
                         ) : <LoadingIndicator isLoading={true}/> }
                     </div>
-                    <div style={{overflow:"auto"}}>
+                    <div style={{overflow:"hidden"}}>
```

Comment:
this fixes scroll issue we talked about yesterday?

Reply:
yeah